demo/demo.py

```diff
@@ -70,6 +72,24 @@ def get_parser():
     return parser
 
 
+def test_opencv_video_format(codec, file_ext):
+    filename = "test_file" + file_ext
```

Comment:
It should be inside a temporary directory (use `tempfile.TemporaryDirectory`). then also remove `os.remove`

Reply:
Sure, will do that.